itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -74,7 +74,7 @@
 /**
  * TestReplicationScenariosAcidTables - test replication for ACID tables.
  */
-@org.junit.Ignore("HIVE-25267")
+//@org.junit.Ignore("HIVE-25267")
```

Comment:
Does this fix the flakiness issue mentioned in HIVE-25267?

Reply:
Thanks for the  review @pvary . No, it doesn't. I wanted to get a feedback on this. Also, the current test were in this file only so had enabled it. I will revert this if I couldn't fix the flakiness as a part of JIRA